Firebase/android/core/src/main/java/com/novoda/bonfire/analytics/ErrorLogger.java

```diff
@@ -0,0 +1,7 @@
+package com.novoda.bonfire.analytics;
+
+public interface ErrorLogger {
+
+    void reportError(String message, Throwable throwable);
```

Comment:
The trend on this is to put `Throwable` in the 1st param. 
It is because if you want to convert the message into a format with additional varags, the last parameter will be `Object...`